docker/emptydrops/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build -t quay.io/humancellatlas/secondary-analysis-dropletutils:0.1.0 .
```

Comment:
I'd make it more flexible and change the script name explicitly to `build_docker.sh`:

```suggestion
tag=$1

if [ -z $tag ]; then
    echo -e "\nYou must provide a tag"
    echo -e "\nUsage: bash build_docker.sh TAG\n"
    exit 1
fi

docker build -t quay.io/humancellatlas/secondary-analysis-dropletutils:$tag .
```